moduleroot/renovate.json.erb

```diff
@@ -1,6 +1,8 @@
 {
   "extends": [
-    "config:base"
+    "config:base",
+    ":gitSignOff",
```

Comment:
I don't think we actually need gitSignOff

Reply:
Yes, we don't need it. But I see no harm either 🤷